src/components/calcite-dropdown/calcite-dropdown.e2e.ts

```diff
@@ -897,4 +897,76 @@ describe("calcite-dropdown", () => {
     await page.waitForChanges();
     expect(await dropdownWrapper.isVisible()).toBe(false);
   });
+
+  it("correct role and aria properties are applied based on selection type", async () => {
```

Comment:
Can you use the `accessible` test helper to check role/aria-attrs instead of manually testing?